viewer/packages/utilities/src/customObject/ICustomObject.ts

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { Object3D, Box3 } from 'three';
+import { CustomObjectIntersection } from './CustomObjectIntersection';
+import { CustomObjectIntersectInput } from './CustomObjectIntersectInput';
+
+/**
+ * This interface encasulate a Object3D, and made it possible to add flags to it.
+ * It might be extended with more flags in the future.
+ * @beta
+ */
+export interface ICustomObject {
```

Comment:
You don't have to change it now, but we probably want to discuss whether we want to keep the convention where we prefix interfaces with "I". We export some interfaces that don't do this, but we should be consistent. Anyway, let's discuss this when these features are stabilized

Reply:
Well. This is a very common way of naming.